tests/test_market.py

```diff
@@ -0,0 +1,423 @@
+        """Test market trades result in the expected wallet balances"""
+        simulator = self.set_up_test(agent_policies=[agent_policy])
+        agent = simulator.agents[1]
+        market_deltas, agent_deltas = simulator.market.open_long(
+            wallet_address=1,
+            trade_amount=agent.amount_to_trade,  # in base: that's the thing in your wallet you want to sell
+        )
+        # peek inside the agent's wallet and see how many bonds they have
+        amount_of_bonds_purchased = agent_deltas.longs[0].balance
+        # sell those bonds to close the long
+        market_deltas, agent_deltas = simulator.market.close_long(
+            mint_time=0,
+            wallet_address=1,
+            trade_amount=amount_of_bonds_purchased,  # in bonds: that's the thing in your wallet you want to sell
+        )
+        actual_deltas = Deltas(market_deltas=market_deltas, agent_deltas=agent_deltas)
+        self.compare_deltas(actual_deltas=actual_deltas, expected_deltas=expected_deltas)
+
+    def run_market_test_close_short(
+        self, agent_policy, expected_deltas: Deltas, delete_logs=True, partial=None, tick_time=None
+    ):  # pylint: disable=too-many-arguments
+        # disabling pylint because this function is used in 3 tests w/ tiny parameter tweaks
+        """Test market trades result in the expected wallet balances"""
+        simulator = self.set_up_test(agent_policies=[agent_policy])
+        agent = simulator.agents[1]
+        market_deltas, agent_deltas = simulator.market.open_short(
+            wallet_address=1,
+            trade_amount=agent.amount_to_trade,  # in bonds: that's the thing you want to short
+        )
+        # peek inside the agent's wallet and see how many bonds they have
+        amount_of_bonds_sold = agent_deltas.shorts[0].balance
+        # sell pthose bonds to close the short (partial amount if selected)
+        trade_amount = amount_of_bonds_sold * partial if partial else amount_of_bonds_sold
+        if tick_time:
+            simulator.market.tick(simulator.market_step_size())
+        market_deltas, agent_deltas = simulator.market.close_short(
+            mint_time=0,
+            wallet_address=1,
+            trade_amount=trade_amount,  # in bonds: that's the thing you owe, and need to buy back
+        )
+        actual_deltas = Deltas(market_deltas=market_deltas, agent_deltas=agent_deltas)
+        self.compare_deltas(actual_deltas=actual_deltas, expected_deltas=expected_deltas)
+        # TODO: do we want to test the final value in the wallet?
+        # we're not actually updating wallets here, but a full simulation test would
+        # we could test the final wallet value there much more easily
+        # self.compare_wallets(actual_wallet=agent.wallet, expected_wallet=expected_wallet)
+        output_utils.close_logging(delete_logs=delete_logs)
+
+
+class MarketTestsOneFunction(BaseMarketTest):
```

Comment:
We have been including the equations that result in expected values (`trade_result`, `fees_paid`) in the test docs. Can you do that here?

Reply:
something like lines 189-195 in test_get_max? since we're testing that elsewhere, i don't see the point of adding it here

Reply:
Just asking for you to note where the numbers came from